third_party/move/move-prover/tests/sources/functional/enum.move

```diff
@@ -0,0 +1,86 @@
+module 0x815::m {
+
+    enum CommonFields has copy, drop {
+        Foo{x: u64, y: u8},
+        Bar{x: u64, y: u8, z: u32}
+    }
+
+    spec CommonFields {
+        invariant self.x > 20;
+        invariant (self is CommonFields::Bar) ==> self.z > 10;
+    }
+
+    // fun t9_common_field(): u64 {
```

Comment:
Why is this function commented out?

Reply:
will remove the comment once #14312 gets landed

Reply:
done